### ISSUE TITLE
Class path contains multiple SLF4J bindings

### ISSUE THREAD

**Issue by benmccann** [playframework/playframework#3206]
[From the mailing list](https://groups.google.com/forum/#!searchin/play-framework/slf4j|sort:date/play-framework/AGpYmpPZLSc/YaMJKcUReLEJ)

> I just switched my app from 2.3.0 to 2.3.2
> 
> I am now getting 
> 
> SLF4J: Class path contains multiple SLF4J bindings.
> SLF4J: Found binding in [jar:file:/Users/jeff/.ivy2/cache/org.slf4j/slf4j-nop/jars/slf4j-nop-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
> SLF4J: Found binding in [jar:file:/Users/jeff/.ivy2/cache/org.slf4j/slf4j-jdk14/jars/slf4j-jdk14-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
> SLF4J: Found binding in [jar:file:/Users/jeff/.ivy2/cache/org.slf4j/slf4j-simple/jars/slf4j-simple-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
> SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
> SLF4J: Actual binding is of type [org.slf4j.helpers.NOPLoggerFactory]

Thought that we had a good workaround for jdeb's sl4j issues, but guess it wasn't enough

[Fixed in jdeb and awaiting a new release](https://github.com/tcurdt/jdeb/pull/174). Then we'll [update sbt-native-packager per the bug there](https://github.com/sbt/sbt-native-packager/issues/291)

**Comment by vader** [comment]
Any idea how to do this with build.scala?

**Comment by dwijnand** [comment]
@benmccann I'm confused. I had identified this issue down to https://github.com/typesafehub/js-engine/issues/44, for which I submitted a (still pending) pull request: https://github.com/typesafehub/js-engine/pull/45. Are you sure this is fixed?

**Comment by benmccann** [comment]
There could be multiple bad actor libraries and we only fixed one of them. Let's get your change to js-engine reviewed and merged. I pinged jroper to look at it, but he's been a bit busy lately, so I'm not sure if he'll respond right away. If not, you might want to ask other folks on the Play developer mailing list or chatroom if they'd be able to help. I'm afraid I haven't had much time to work on Play stuff lately either
